CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+Contributing to Dataspace Connector
+===================================
+
+Thank you for your interest contributing to
+the [Eclipse Dataspace Connector](https://projects.eclipse.org/projects/technology.dataspaceconnector)!
+
+## Table of Contents
```

Comment:
let's check the length of the md in sum. Maybe toc is not needed at all.

Reply:
I personally would like to keep the TOC since it provides orientation to the reader.